Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN cd /src/ \
     && ansible-playbook -vv -c local -i localhost, files/install-deps.yaml \
     && dnf clean all
 
+RUN pip3 install git+https://github.com/packit-service/packit-service
```

Comment:
Why is this needed? I would expect using local dir for that.

Reply:
Oh, that was just a leftover from my local testing to quickly workaround [incompatibility](https://github.com/packit-service/packit/pull/614). Thanks for noticing.

Reply:
Thanks!